scene/animation/animation_player.cpp

```diff
@@ -236,7 +236,12 @@ void AnimationPlayer::_process_playback_data(PlaybackData &cd, double p_delta, f
 	}
 	// AnimationPlayer doesn't have internal seeking.
 	// However, immediately after playback, discrete keys should be retrieved with EXACT mode since behind keys must be ignored at that time.
-	pi.is_external_seeking = !p_started;
+	if (backwards) {
+		pi.is_external_seeking = true;
+	} else {
+		pi.is_external_seeking = !p_started;
+	}
```

Comment:
If this is valid it should just be:
```cpp
pi.is_external_seeking = backwards || !p_started;
```
No need for two blocks and much easier to read